<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>USense</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource.qrc">
    <normaloff>:/images/USenseIcon.png</normaloff>:/images/USenseIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="up_5" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_up_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="up_6" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_up_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="up_11" native="true">
   <property name="geometry">
    <rect>
     <x>944</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_up_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resource.qrc">:/images/关闭.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="up_1" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_up_1">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resource.qrc">:/images/SinaBeforeClicked.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="up_2" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_up_2">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resource.qrc">:/images/RenrenBeforeClicked.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="up_3" native="true">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_up_3">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../Resource.qrc">:/images/AppBeforeClicked.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="up_8" native="true">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_up_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="up_9" native="true">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_up_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="up_7" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_up_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="up_10" native="true">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_up_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="up_4" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_up_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1021</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout"/>
  </widget>
  <widget class="QLabel" name="photoLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="loading_label">
   <property name="geometry">
    <rect>
     <x>528</x>
     <y>284</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
